windows/src/global/delphi/general/KeymanVersion.pas

```diff
@@ -33,6 +33,7 @@ interface
   SKeymanKeyboardVersion = SKeymanVersion100;
```

Comment:
At what point does `SKeymanKeyboardVersion` go to `SKeymanVersion110`?

Reply:
We up the `SKeymanKeyboardVersion` when we get a breaking compat change. No plans for breaking changes in v11 at this stage.